.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -0,0 +1,64 @@
+name: "\U0001F41B Bug report"
+description: "Report a bug with this project."
+labels: "type:bug"
+body:
+    - type: markdown
+      attributes:
+          value: |
+              Thanks for taking the time to fill out this bug report!  Please fill in as much of the template below as you can.
+    - type: textarea
+      attributes:
+          label: Describe the bug
+          description: Please write a clear and concise description of the bug, including what you expect to happen and what is currently happening.
+          placeholder: |
+              Feature '...' is not working properly. I expect '...' to happen, but '...' happens instead
+      validations:
+          required: true
+
+    - type: textarea
+      attributes:
+          label: Steps to Reproduce
+          description: Please write the steps needed to reproduce the bug.
+          placeholder: |
+              1. Go to '...'
+              2. Click on '...'
+              3. Scroll down to '...'
+              4. See error
+      validations:
+          required: true
+
+    - type: textarea
+      attributes:
+          label: Screenshots, screen recording, code snippet
+          description: |
+              If possible, please upload a screenshot or screen recording which demonstrates the bug. You can use LIEcap to create a GIF screen recording: https://www.cockos.com/licecap/
+              Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+              For small snippets paste it directly here, or you can use GitHub Gist to share multiple code files: https://gist.github.com
+              Please ensure the shared code can be used by a developer to reproduce the issue—ideally it can be copied into a local development environment or executed in a browser console to help debug the issue
+      validations:
+          required: false
+          
+    - type: textarea
+      attributes:
+          label: Environment information
```

Comment:
Would it be possible to expand this area by several additional blocks: Agent OS, Agent/Server/UI Server version, Browser version?
Also in case of "module" issue it would be helpful to get module code (export from the UI).

Reply:
Aread removed